doc/source/serve/multi-app.md

```diff
@@ -0,0 +1,214 @@
+
+![replica-overview](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/multi-app/replica-overview-dashboard.png)
+
+## New Multi-Application Config
+
+Use the config from the above tutorial as an example. In a multi-application config, the first section is for cluster-level config options:
+```yaml
+proxy_location: EveryNode
+http_options:
+  host: 0.0.0.0
+  port: 8000
+```
+
+Then, specify a list of applications to deploy to the Ray cluster. Each application must have a unique name and route prefix.
+```yaml
+applications:
+
+- name: app1
+  route_prefix: /calculator
+  import_path: calculator:app
+  runtime_env: {}
+  deployments:
+  - name: Multiplier
+  - name: Adder
+  - name: Router
+  - name: DAGDriver
+
+- name: app2
+  route_prefix: /greet
+  import_path: greet:app
+  runtime_env: {}
+  deployments:
+  - name: greet
+  - name: DAGDriver
+```
+
+(serve-config-migration)=
+### Migrating from a Single-Application Config
+
+Migrating the single-application config `ServeApplicationSchema` to the multi-application config format `ServeDeploySchema` is straightforward. Each entry under the  `applications` field matches the old, single-application config format. To convert a single-application config to the multi-application config format:
+* Copy the entire old config to an entry under the `applications` field.
+* Remove `host` and `port` from the entry and move them under the `http_options` field.
+* Name the application.
+* If you haven't already, set the application-level `route_prefix` to the route prefix of the ingress deployment in the application.
+* When needed, add more applications.
+
+For more details on the multi-application config format, see the documentation for [`ServeDeploySchema`](serve-rest-api-config-schema).
+
+:::{note} 
+You must remove `host` and `port` from the application entry. In a multi-application config, specifying cluster-level options within an individual application isn't applicable, and is not supported.
+:::
+
+## One application vs Multiple Applications
+It can be useful to deploy multiple independent applications on one Ray cluster. However, it is also important to keep in mind the downsides of doing so.
+
+We suggest you deploy multiple applications per cluster if:
+* You want to save cost by sharing resources between applications.
+* You have many small independent models, and you want to be able to easily add and delete models often.
+
+However, we suggest you limit to one application per cluster if:
+* Your applications are large and require a lot of resources. Deploying multiple applications per cluster can cause competition of resources in this case.
+* You want better isolation between applications. Although applications are independent of one another, isolation can't be guaranteed. Also, if you need to change the configuration of the cluster for one application, all other applications will also be affected. 
```

Comment:
I think it would be helpful to add a section on how deployments will work for adding/removing/updating applications